src/js/structs/PodSpec.js

```diff
@@ -0,0 +1,53 @@
+import ServiceSpec from './ServiceSpec';
+
+module.exports = class PodSpec extends ServiceSpec {
+  getLabels() {
+    return this.get('labels');
+  }
+
+  getContainers() {
+    return this.get('containers') || [];
+  }
+
+  getContainerCount() {
+    return this.getContainers().length;
+  }
+
+  getResourcesSummary() {
+    return this.getContainers().reduce(function (resources, container) {
+
+      Object.keys(container.resources).forEach(function (key) {
+        resources[key] += container.resources[key];
+      });
+
+      return resources;
+    }, {
+      cpus: 0,
+      mem: 0,
+      gpus: 0,
+      disk: 0
+    });
+  }
+
+  getContainerSpec(name) {
+    let matches = this.getContainers().filter(function (container) {
```

Comment:
You could do this `let [spec = null] =  this.getContainers().filter...` instead of all the conditions #nit